CMSgov/mint-app
# NOREF Update Operational Need Triggers

## Changes and Description

- Correct triggers for
    - Update the plan’s contract
    - Educate beneficiaries
- Calculate the if needs are needed again for all existing needs based on question answers.

## How to test this change

1. Bring up the App and make a Model.
2.  Answer the question   ` Will the model have a learning strategy?` in plan ops, eval, and learning, with any below answer ( or combination) and ensure the need is shown as needed
    - `We plan to educate beneficiaries`
    - `We plan to use an IT platform (Connect)`
    - `We plan to have a learning contractor (cross-model or individual)`
    - `We plan to enable participant-to-participant collaboration`
    - `Other`
3. Answer the question   `Have you updated the plan’s contract?` in general characteristics with yes and ensure the need is shown as needed



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
